src/test/test_addr.c

```diff
@@ -1671,6 +1671,128 @@ test_addr_octal(void *arg)
   ;
 }
 
+#define Get_IPv4(test_addr, str) STMT_BEGIN                     \
```

Comment:
We name function-like macros using snake_case:
```suggestion
#define get_ipv4(test_addr, str) STMT_BEGIN                     \
```